pulsar-flink-connector/src/main/java/org/apache/flink/streaming/connectors/pulsar/internal/PulsarOptions.java

```diff
@@ -55,7 +55,7 @@
     public static final String MAX_BLOCK_TIME_MS = "max-block-time-ms";
     public static final String POLL_TIMEOUT_MS_OPTION_KEY = "poll-timeout-ms";
     public static final String SEND_TIMEOUT_MS = "send-timeout-ms";
-    public static final String SUBSCRIPTION_ROLE_OPTION_KEY = "subscription-role-prefix";
+    public static final String SUBSCRIPTION_ROLE_OPTION_KEY = "subscriptionRolePrefix";
```

Comment:
could you keep it consistent with other options here?

Reply:
the pulsar reader  config now seems changes to subscriptionRolePrefix, if keep it 'subscription-role-prefix', exception throws like 
![image](https://user-images.githubusercontent.com/37431499/141036195-72f360d3-3a55-4302-bb06-cf4bc6daab71.png)
from the log I see this config show be 'subscriptionRolePrefix'

Reply:
Thanks for the clarification. These config inconsistencies are causing a lot of confusions.

We will see how to work on cleaning them later

Reply:
OK